### PR TITLE
Erikslud explore wicket bootstrap

### PR DESCRIPTION
**Pull request by erikslud**

### CODE REVIEW

**Comment by erikslud** [comment]
More bootstrap adjustment needed to get a robust and consistent "look and feel". This is just a first step.

File: WebWickerCore/src/main/java/com/tieto/webwicker/web/HomePage.html [hunk 1]
```diff
@@ -1,35 +1,46 @@
...
+
+			<h1 wicket:id="title">WebWicker</h1>
+
+			<div id="menu"></div>
+		</div>
+		<div class="row">
+			<div class="navbar-inverse">
+				<ul class ="list-inline">
+
+					<li>
```
Comment (svebichr) [inline WebWickerCore/src/main/java/com/tieto/webwicker/web/HomePage.html]: Duplicate li element.

**Review by svebichr — CHANGES_REQUESTED** [review]
I think you may have a bad autocrlf setting since all files you edited has been converted to crlf instead of the normalized lf. This makes it very difficult to review, it looks like everything is changed.

File: WebWickerEiffel/src/main/java/com/tieto/webwicker/eiffel/web/commits/CommitPage.java [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
-	private static final List<String> partlyApproved = Arrays.asList(new String[]{"+1"});
-	private static final List<String> notApproved = Arrays.asList(new String[]{"-1","-2"});
-	private static final List<String> verified = Arrays.asList(new String[]{"+1"});
-	private static final List<String> notVerified = Arrays.asList(new String[]{"-1"});
+	
+	private static final List<String> approved = Arrays.asList(new String[]{"SUCCESS"});
+	private static final List<String> partlyApproved = Arrays.asList(new String[]{"INCONCLUSIVE"});
+	private static final List<String> notApproved = Arrays.asList(new String[]{"FAILURE"});
+	private static final List<String> verified = Arrays.asList(new String[]{"SUCCESS"});
+	private static final List<String> notVerified = Arrays.asList(new String[]{"INCONCLUSIVE"});
```
Comment (svebichr) [inline WebWickerEiffel/src/main/java/com/tieto/webwicker/eiffel/web/commits/CommitPage.java]: Shouldn't it be INCONCLUSIVE and FAILURE? Or can FAILURE never occur for verified?

**Pull request merged**